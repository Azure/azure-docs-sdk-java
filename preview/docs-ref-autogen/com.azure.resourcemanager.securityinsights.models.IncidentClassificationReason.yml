### YamlMime:JavaType
uid: "com.azure.resourcemanager.securityinsights.models.IncidentClassificationReason"
fullName: "com.azure.resourcemanager.securityinsights.models.IncidentClassificationReason"
name: "IncidentClassificationReason"
nameWithType: "IncidentClassificationReason"
summary: "The classification reason the incident was closed with."
inheritances:
- "<xref href=\"java.lang.Object?displayProperty=fullName\" data-throw-if-not-resolved=\"False\" />"
- "<xref href=\"com.azure.core.util.ExpandableStringEnum?displayProperty=fullName\" data-throw-if-not-resolved=\"False\" />"
inheritedClassMethods:
- classRef: "<xref href=\"com.azure.core.util.ExpandableStringEnum?alt=com.azure.core.util.ExpandableStringEnum&text=ExpandableStringEnum\" data-throw-if-not-resolved=\"False\" />"
  methodsRef:
  - "<xref href=\"com.azure.core.util.ExpandableStringEnum.<T>fromString(java.lang.String,java.lang.Class<T>)?alt=com.azure.core.util.ExpandableStringEnum.<T>fromString&text=<T>fromString\" data-throw-if-not-resolved=\"False\" />"
  - "<xref href=\"com.azure.core.util.ExpandableStringEnum.<T>values(java.lang.Class<T>)?alt=com.azure.core.util.ExpandableStringEnum.<T>values&text=<T>values\" data-throw-if-not-resolved=\"False\" />"
  - "<xref href=\"com.azure.core.util.ExpandableStringEnum.equals(java.lang.Object)?alt=com.azure.core.util.ExpandableStringEnum.equals&text=equals\" data-throw-if-not-resolved=\"False\" />"
  - "<xref href=\"com.azure.core.util.ExpandableStringEnum.hashCode()?alt=com.azure.core.util.ExpandableStringEnum.hashCode&text=hashCode\" data-throw-if-not-resolved=\"False\" />"
  - "<xref href=\"com.azure.core.util.ExpandableStringEnum.toString()?alt=com.azure.core.util.ExpandableStringEnum.toString&text=toString\" data-throw-if-not-resolved=\"False\" />"
- classRef: "java.lang.<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html\">Object</a>"
  methodsRef:
  - "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#clone--\">clone</a>"
  - "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#finalize--\">finalize</a>"
  - "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#getClass--\">getClass</a>"
  - "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#notify--\">notify</a>"
  - "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#notifyAll--\">notifyAll</a>"
  - "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#wait--\">wait</a>"
  - "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#wait-long-\">wait</a>"
  - "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#wait-long-int-\">wait</a>"
syntax: "public final class IncidentClassificationReason extends ExpandableStringEnum<IncidentClassificationReason>"
constructors:
- uid: "com.azure.resourcemanager.securityinsights.models.IncidentClassificationReason.IncidentClassificationReason()"
  fullName: "com.azure.resourcemanager.securityinsights.models.IncidentClassificationReason.IncidentClassificationReason()"
  name: "IncidentClassificationReason()"
  nameWithType: "IncidentClassificationReason.IncidentClassificationReason()"
  syntax: "public IncidentClassificationReason()"
fields:
- uid: "com.azure.resourcemanager.securityinsights.models.IncidentClassificationReason.INACCURATE_DATA"
  fullName: "com.azure.resourcemanager.securityinsights.models.IncidentClassificationReason.INACCURATE_DATA"
  name: "INACCURATE_DATA"
  nameWithType: "IncidentClassificationReason.INACCURATE_DATA"
  summary: "Static value Inaccurate<wbr>Data for Incident<wbr>Classification<wbr>Reason."
  modifiers:
  - "static"
  - "final"
  field:
    type: "<xref href=\"com.azure.resourcemanager.securityinsights.models.IncidentClassificationReason?alt=com.azure.resourcemanager.securityinsights.models.IncidentClassificationReason&text=IncidentClassificationReason\" data-throw-if-not-resolved=\"False\" />"
  syntax: "public static final IncidentClassificationReason INACCURATE_DATA"
  desc: "Static value InaccurateData for IncidentClassificationReason."
- uid: "com.azure.resourcemanager.securityinsights.models.IncidentClassificationReason.INCORRECT_ALERT_LOGIC"
  fullName: "com.azure.resourcemanager.securityinsights.models.IncidentClassificationReason.INCORRECT_ALERT_LOGIC"
  name: "INCORRECT_ALERT_LOGIC"
  nameWithType: "IncidentClassificationReason.INCORRECT_ALERT_LOGIC"
  summary: "Static value Incorrect<wbr>Alert<wbr>Logic for Incident<wbr>Classification<wbr>Reason."
  modifiers:
  - "static"
  - "final"
  field:
    type: "<xref href=\"com.azure.resourcemanager.securityinsights.models.IncidentClassificationReason?alt=com.azure.resourcemanager.securityinsights.models.IncidentClassificationReason&text=IncidentClassificationReason\" data-throw-if-not-resolved=\"False\" />"
  syntax: "public static final IncidentClassificationReason INCORRECT_ALERT_LOGIC"
  desc: "Static value IncorrectAlertLogic for IncidentClassificationReason."
- uid: "com.azure.resourcemanager.securityinsights.models.IncidentClassificationReason.SUSPICIOUS_ACTIVITY"
  fullName: "com.azure.resourcemanager.securityinsights.models.IncidentClassificationReason.SUSPICIOUS_ACTIVITY"
  name: "SUSPICIOUS_ACTIVITY"
  nameWithType: "IncidentClassificationReason.SUSPICIOUS_ACTIVITY"
  summary: "Static value Suspicious<wbr>Activity for Incident<wbr>Classification<wbr>Reason."
  modifiers:
  - "static"
  - "final"
  field:
    type: "<xref href=\"com.azure.resourcemanager.securityinsights.models.IncidentClassificationReason?alt=com.azure.resourcemanager.securityinsights.models.IncidentClassificationReason&text=IncidentClassificationReason\" data-throw-if-not-resolved=\"False\" />"
  syntax: "public static final IncidentClassificationReason SUSPICIOUS_ACTIVITY"
  desc: "Static value SuspiciousActivity for IncidentClassificationReason."
- uid: "com.azure.resourcemanager.securityinsights.models.IncidentClassificationReason.SUSPICIOUS_BUT_EXPECTED"
  fullName: "com.azure.resourcemanager.securityinsights.models.IncidentClassificationReason.SUSPICIOUS_BUT_EXPECTED"
  name: "SUSPICIOUS_BUT_EXPECTED"
  nameWithType: "IncidentClassificationReason.SUSPICIOUS_BUT_EXPECTED"
  summary: "Static value Suspicious<wbr>But<wbr>Expected for Incident<wbr>Classification<wbr>Reason."
  modifiers:
  - "static"
  - "final"
  field:
    type: "<xref href=\"com.azure.resourcemanager.securityinsights.models.IncidentClassificationReason?alt=com.azure.resourcemanager.securityinsights.models.IncidentClassificationReason&text=IncidentClassificationReason\" data-throw-if-not-resolved=\"False\" />"
  syntax: "public static final IncidentClassificationReason SUSPICIOUS_BUT_EXPECTED"
  desc: "Static value SuspiciousButExpected for IncidentClassificationReason."
methods:
- uid: "com.azure.resourcemanager.securityinsights.models.IncidentClassificationReason.fromString(java.lang.String)"
  fullName: "com.azure.resourcemanager.securityinsights.models.IncidentClassificationReason.fromString(String name)"
  name: "fromString(String name)"
  nameWithType: "IncidentClassificationReason.fromString(String name)"
  summary: "Creates or finds a Incident<wbr>Classification<wbr>Reason from its string representation."
  modifiers:
  - "static"
  parameters:
  - description: "a name to look for."
    name: "name"
    type: "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/String.html\">String</a>"
  syntax: "public static IncidentClassificationReason fromString(String name)"
  desc: "Creates or finds a IncidentClassificationReason from its string representation."
  returns:
    description: "the corresponding IncidentClassificationReason."
    type: "<xref href=\"com.azure.resourcemanager.securityinsights.models.IncidentClassificationReason?alt=com.azure.resourcemanager.securityinsights.models.IncidentClassificationReason&text=IncidentClassificationReason\" data-throw-if-not-resolved=\"False\" />"
- uid: "com.azure.resourcemanager.securityinsights.models.IncidentClassificationReason.values()"
  fullName: "com.azure.resourcemanager.securityinsights.models.IncidentClassificationReason.values()"
  name: "values()"
  nameWithType: "IncidentClassificationReason.values()"
  summary: "Gets known Incident<wbr>Classification<wbr>Reason values."
  modifiers:
  - "static"
  syntax: "public static Collection<IncidentClassificationReason> values()"
  desc: "Gets known IncidentClassificationReason values."
  returns:
    description: "known IncidentClassificationReason values."
    type: "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/util/Collection.html\">Collection</a>&lt;<xref href=\"com.azure.resourcemanager.securityinsights.models.IncidentClassificationReason?alt=com.azure.resourcemanager.securityinsights.models.IncidentClassificationReason&text=IncidentClassificationReason\" data-throw-if-not-resolved=\"False\" />&gt;"
type: "class"
desc: "The classification reason the incident was closed with."
metadata: {}
package: "com.azure.resourcemanager.securityinsights.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-securityinsights:1.0.0-beta.4
