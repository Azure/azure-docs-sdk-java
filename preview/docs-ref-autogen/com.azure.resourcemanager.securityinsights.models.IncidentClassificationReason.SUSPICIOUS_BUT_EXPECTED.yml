### YamlMime:JavaMember
uid: "com.azure.resourcemanager.securityinsights.models.IncidentClassificationReason.SUSPICIOUS_BUT_EXPECTED*"
fullName: "com.azure.resourcemanager.securityinsights.models.IncidentClassificationReason.SUSPICIOUS_BUT_EXPECTED"
name: "SUSPICIOUS_BUT_EXPECTED"
nameWithType: "IncidentClassificationReason.SUSPICIOUS_BUT_EXPECTED"
members:
- uid: "com.azure.resourcemanager.securityinsights.models.IncidentClassificationReason.SUSPICIOUS_BUT_EXPECTED"
  fullName: "com.azure.resourcemanager.securityinsights.models.IncidentClassificationReason.SUSPICIOUS_BUT_EXPECTED"
  name: "SUSPICIOUS_BUT_EXPECTED"
  nameWithType: "IncidentClassificationReason.SUSPICIOUS_BUT_EXPECTED"
  summary: "Static value SuspiciousButExpected for IncidentClassificationReason."
  syntax: "public static final IncidentClassificationReason SUSPICIOUS_BUT_EXPECTED"
type: "field"
metadata: {}
package: "com.azure.resourcemanager.securityinsights.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-securityinsights:1.0.0-beta.1
